### DISCUSSION TITLE
Update to Node.js 18

### DISCUSSION THREAD

**Discussion by ComradeVanti** [openupm/openupm-cli#282] [Ideas]
Seems like Node.js 16 has been out of maintenance [for a while now](https://nodejs.org/en/about/previous-releases).

We could and maybe should update to 18. For new features and security stuff.

But first I wanted to open this discussion to see if there are any objections.

**Comment by favoyang** [comment]
It's okay.

We have a few places defined `v16` that need to be updated
- .nvmrc
- volta in package.json
- .node-version
- .github/workflows/ci.yml

**Reply by ComradeVanti** [reply]
Got it, thanks

**Comment by ComradeVanti** [comment]
Leaving this open for a while longer in case anyone else wants to chime in on this.

**Comment by ComradeVanti** [comment]
There is now a PR for this here: https://github.com/openupm/openupm-cli/pull/315

**Comment by ComradeVanti** [comment]
PR is now merged.
